pcdsdevices/ccm.py

```diff
@@ -257,8 +257,24 @@ def _state(self, value):
 
 # Calculations between alio position and energy, with all intermediates.
 def theta_to_alio(theta, theta0, gr, gd):
-    """Converts theta angle (rad) to alio position (mm)."""
-    return gr * (1/np.cos(theta)-1) + gd * np.tan(theta - theta0)
+    """
+    Converts theta angle (rad) to alio position (mm).
+
+    Theta_B:       scattering angle, the angle reduces when rotating clockwise
+                   (Bragg angle)
+    Theta_0:       scattering angle offset of the Si (111) first crystal
+    Delta_Theta:   the effective scattering angle (adjusted with Alio stage)
+    R = 0.003175m: radius of the sapphire ball connected to the Alio stage
+    D = 0.232156m: distance between the Theta_B rotation axis and the center
```

Comment:
Based on the paper https://docs.google.com/viewer?a=v&pid=sites&srcid=ZGVmYXVsdGRvbWFpbnx4cHB3aWtpfGd4OjQ0ZDA5ZGFkYTM0MjFhZDk the `D` value is slightly different - I did not change the number, but I should probably change the docstring here to reflect the actual number we are using - out of curiosity though - did this change because we have new hardware/devices?  
I got the paper from (https://sites.google.com/site/xppwiki/beamline-components/ccm)

Reply:
It may be worth reaching out to the requester to see why the constant is different, and if it's for a good reason, documenting it here.